packages/stark-ui/assets/styles/_header.scss

```diff
@@ -129,7 +129,6 @@
 
 /* Temporary classes to simulate components that have not been implemented yet */
 .app-logo {
```

Comment:
Now that we use the `AppLogo` component, these CSS rules are not needed anymore right? The ones from the component should take effect instead...